.eslintrc.json

```diff
@@ -10,48 +10,53 @@
     "next",
```

Comment:
If you want to change the eslintrc let me know first. I'm pretty sure the configs work fine.